### PR TITLE
fixed edit category issues using async await, and disabled save butto…

### PR DESCRIPTION
**Pull request by natalievasquez11**
…n to prevent unnecessary API calls

### CODE REVIEW

File: src/components/dashboard/Dashboard.js [hunk 1]
```diff
@@ -12,8 +12,10 @@ class Dashboard extends Component {
         categories: [],
         chores: [],
         categoryName: '',
-        users: []
+        users: [],
+        editSaveBtn: true
     }
+    OGCategoryName = '';
```
Comment (mattvella07) [inline src/components/dashboard/Dashboard.js:18]: Cool name!

File: src/components/dashboard/Dashboard.js [hunk 2]
```diff
@@ -12,8 +12,10 @@ class Dashboard extends Component {
         categories: [],
         chores: [],
         categoryName: '',
-        users: []
+        users: [],
+        editSaveBtn: true
```
Comment (mattvella07) [inline src/components/dashboard/Dashboard.js:16]: I think the variable name should be changed to make it easier to understand when you come back to the code later.  Maybe something like `editSaveBtnDisabled`

File: src/components/dashboard/Dashboard.js [hunk 3]
```diff
@@ -169,59 +172,49 @@ class Dashboard extends Component {
...
-                        this.getCategories();
-                    })
-                    .catch(err => {
-                        console.log(err);
-                    })
-                    this.setState({ 
-                        categoryName: categoryName,
-                        users: userIdArr
-                    });
+        if(this.OGCategoryName !== categoryName) {
```
Comment (mattvella07) [inline src/components/dashboard/Dashboard.js:196]: Does it work if you only change a user and not the category name?

Comment (natalievasquez11) [reply src/components/dashboard/Dashboard.js]: I changed it, if you want to look at it again.

**Review by mattvella07 — APPROVED** [review]

**Pull request merged**